clang/runtime/dpct-rt/include/kernel.hpp.inc

```diff
@@ -86,5 +126,237 @@ static kernel_function_info get_kernel_function_info(const void *function) {
 }
 // DPCT_LABEL_END
 
+namespace detail {
+
+static std::vector<char> read_file_to_vector(const std::string &name) {
```

Comment:
Why can't we use std::string as the return type?

Reply:
Function is no longer needed.